dashboard/js/app/templates/main.html

```diff
@@ -91,6 +97,13 @@
                 <div id="registeredDriversInst"></div>
             </div>
 
+            <div class="tab-pane alvrSection fade" id="debug">
+                <br />
+                <button id="captureInputFrame" type="button" class="btn btn-primary"> <%= captureInputFrame%> </button>
+                <button id="captureOutputFrame" type="button" class="btn btn-primary"> <%= captureOutputFrame%> </button>
+                <input id="captureFramePath" type="text" value="/tmp/alvr-capture.ppm"> </input>
```

Comment:
Use a setting entry for the path. for the GET requests you don't have to pass any body, or you could make a single request "/api/capture" with body "input" or "output" strings.

Reply:
if you want to os-gate the display the button you can get the os name from /api/server-os, or just leave the buttons non-functional on windows, but make sure they don't crash the server. As a last note, don't worry too much about the dashboard since it will be replaced.